.github/CODEOWNERS

```diff
@@ -0,0 +1,28 @@
+# GitHub CODEOWNERS definition
+# See: https://help.github.com/articles/about-codeowners/
+
+* @elastic/beats
+
+/auditbeat/module/ @elastic/secops
+/x-pack/auditbeat/ @elastic/secops
+
+/packetbeat/protos/ @elastic/secops
+/x-pack/packetbeat/ @elastic/secops
+
+/metricbeat/module/ @elastic/infrastructure
```

Comment:
could we add a substructure for stack monitoring for the Elastic Stack modules?

Reply:
done